wp-content/civi-extensions/civirazorpay/api/v3/Civirazorpay/CivicrmRazorpayFetchSettlementDateCron.php

```
@@ -0,0 +1,355 @@
+    sleep(pow(2, $this->retryCount));
+    $this->run();
+  }
+
+}
+
+/**
+ * Define the API specification.
+ *
+ * @param array $spec
+ */
+function _civicrm_api3_civirazorpay_civicrmrazorpayfetchsettlementdatecron_spec(&$spec) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$spec'. (Unused Code Rules)

(UnusedFormalParameter)